directories/src/utils/datamap.js

```diff
@@ -2,14 +2,31 @@ import _ from "lodash";
 import iota from "redux/services/iota";
 import Encryption from "utils/encryption";
 
-const generate = (handle, size) => {
+// const generate = (handle, size) => {
+// const keys = _.range(0, size + 1);
+
+// const [dataMap, _hash] = _.reduce(
+// keys,
+// ([dataM, hash], i) => {
+// const [obfuscatedHash, nextHash] = Encryption.hashChain(hash);
+// dataM[i] = iota.toAddress(iota.utils.toTrytes(obfuscatedHash));
+
+// return [dataM, nextHash];
+// },
+// [{}, Encryption.genesisHash(handle)]
+// );
+
+// return dataMap;
+// };
```

Comment:
the webnode needs a variation of this, we don't want to recreate the datamap for each chunk since it is memory intensive.

the webnode needs a way to pass / save the raw datamap value before it is saved because we can derive the following datamap hashes from it.

Reply:
As an FYI, Jet might be pulling this datamap generation into the IXI module, so we could potentially call it with something like iota.Oyster.generateDataMap(params).  Not sure how far along the changes are.